### ISSUE TITLE
errbot --init produces a traceback

### ISSUE THREAD

**Issue by gutman2019** [errbotio/errbot#1348]
In order to let us help you better, please fill out the following fields as best you can:

### I am...

* [X] Reporting a bug
* [ ] Suggesting a new feature
* [ ] Requesting help with running my bot
* [ ] Requesting help writing plugins
* [ ] Here about something else

### I am running...

* Errbot version: latest
* OS version: Ubuntu 18.04.2 
* Python version: 3.6 
* Using a virtual environment: yes

### Issue description

traceback in various files under 

Traceback (most recent call last):
  File "/home/greg/errbot/bin/errbot", line 6, in <module>
    from errbot.cli import main
  File "/home/greg/errbot/lib/python3.6/site-packages/errbot/__init__.py", line 15, in <module>
    from . import core
  File "/home/greg/errbot/lib/python3.6/site-packages/errbot/core.py", line 30, in <module>
    from .templating import tenv
  File "/home/greg/errbot/lib/python3.6/site-packages/errbot/templating.py", line 3, in <module>
    from errbot.plugin_info import PluginInfo
  File "/home/greg/errbot/lib/python3.6/site-packages/errbot/plugin_info.py", line 5, in <module>
    from dataclasses import dataclass
ModuleNotFoundError: No module named 'dataclasses'

### Steps to reproduce

install virtualenv
initialize virtualenv
pip install errbot
errbot --init

### Additional info
none

**Comment by sijis** [comment]
You will need to install the `dataclasses` library. Once that's done it will work
```
...
ModuleNotFoundError: No module named 'dataclasses'
root@05e68154d0c3:/# pip install dataclasses
Collecting dataclasses
  Downloading https://files.pythonhosted.org/packages/26/2f/1095cdc2868052dd1e64520f7c0d5c8c550ad297e944e641dbf1ffbb9a5d/dataclasses-0.6-py3-none-any.whl
Installing collected packages: dataclasses
Successfully installed dataclasses-0.6
root@05e68154d0c3:/# errbot --init
Your Errbot directory has been correctly initialized !
Just do "errbot" and it should start in text/development mode.
root@05e68154d0c3:/# 
```

We do have a [condition in our setup.py](https://github.com/errbotio/errbot/blob/master/setup.py#L47-L48) to handle this scenario, so its likely a bug there.

**Comment by sijis** [comment]
I've also realized you can also workaround this bug by installing the source distribution instead of the wheel.
`$ pip3 install errbot --no-binary errbot`

A fix should go out tomorrow where the workaround should not be needed.

Thank you for reporting the problem.

**Comment by sijis** [comment]
Fix should be available in 6.1.1.
